test/src/test/java/org/corfudb/runtime/object/CompileProxyTest.java

```diff
@@ -40,6 +44,30 @@ public void testObjectMapSimple() throws Exception {
                 .containsEntry("hell", "world");
     }
 
+    @Test
+    public void testTrimmedObject() throws Exception {
```

Comment:
comments

Reply:
There are comments inline. 

// Advance the sequencer counter and trim up to the tail, so that the first
        // read will be on a trimmed address